### PR TITLE
automake: update to 1.16.4

### PR DESCRIPTION
**Pull request by FnControlOption**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS x.y
Xcode x.y

###### Verification 
Have you

- [ ] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [ ] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [ ] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [ ] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

**Review by jmroot — CHANGES_REQUESTED** [review]
`patch-issue57329.diff` should be removed entirely since the issue has been resolved upstream.

**Review by jmroot — APPROVED** [review]

**Comment by jmroot** [comment]
@herbygillot There was still about half a day to go before maintainer timeout. Please don't merge before the full time is up if the maintainer has not given approval.

**Comment by herbygillot** [comment]
> @herbygillot There was still about half a day to go before maintainer timeout. Please don't merge before the full time is up if the maintainer has not given approval.

Acknowledged, my mistake.

**Pull request merged**
